package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "reactmentoring",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "webpack-dev": "webpack-dev-server --debug --hot --devtool eval-source-map --output-pathinfo --watch --colors --inline --content-base public --port 8090 --host 0.0.0.0",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AndreyKirshin/ReactMentoring.git"
+  },
+  "author": "Andrei Kirshin",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AndreyKirshin/ReactMentoring/issues"
+  },
+  "homepage": "https://github.com/AndreyKirshin/ReactMentoring#readme",
+  "dependencies": {
+    "body-parser": "^1.12.3",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4"
+  },
+  "devDependencies": {
```

Comment:
Why these dependencies?
Let's use the latest babel and webpack (babel 7+, webpack 4+) since we use latest react?

Reply:
done

Reply:
Please take a look at babel 7
https://babeljs.io/setup#installation

Reply:
will be done in next task